MatomoTracker/MatomoTracker.swift

```diff
@@ -399,6 +399,25 @@ extension MatomoTracker {
             dimension.index != index
         })
     }
+    
+    /// remove all without ignore
+    @objc public func removeAll(ignoreIndexs indexs: [NSNumber]? = nil) {
+        if let array = indexs, array.count > 0 {
+            
+            let tempArray = dimensions
+            dimensions.removeAll()
+            for i in array {
```

Comment:
The time complexity of this implementation is not great, O(n2)
(probably not a big practical problem since the array will not contain that many element)